### PR TITLE
Expose internal names

### PR DESCRIPTION
**Pull request by viennaa**
For crossmapping in other tools it is helpful to expose the internal VMWare name of components. All components which have an internal name will now have the additional attribute .internal_name and the inventory will shout it out.

We need this for a cross-collaboration project to map the respective internal names to the real world. They will ask the inventory.

### CODE REVIEW

File: tools/Vrops.py [hunk 1]
```diff
@@ -128,6 +128,13 @@ def get_resources(self, target: str, token: str,
                     resource_object.uuid = resource["resource"]["identifier"]
                     resource_object.resourcekind = resource["resource"]["resourceKey"]["resourceKindKey"]
                     resource_object.parent = resource.get("relatedResources", [None])[0]
+                    resource_identifiers = resource.get('resource').get('resourceKey').get('resourceIdentifiers')
```
Comment (richardtief) [inline tools/Vrops.py:131]: This might fail because the second and the third `.get()` is executed on a `None`. You should provide the alternative.

**Review by richardtief — CHANGES_REQUESTED** [review]
Good catch. Didn't expect to find the info in the api. I have seen one small problem.

File: tools/Vrops.py (hunk 1 above)
Comment (viennaa) [reply tools/Vrops.py]: Absolutely right, corrected.

**Review by richardtief — APPROVED** [review]
Thanks. Let's open the inventory for other services.

**Pull request merged**